sass/spinner.scss

```diff
@@ -0,0 +1,81 @@
+$color: #fff;
```

Comment:
We don't. I think it's a better approach to do this in with CSS animations, though.